lib/api/objectCopy.js

```diff
@@ -351,6 +356,31 @@ function objectCopy(authInfo, request, sourceBucket,
                     serverSideEncryption, destBucketMD);
             }
 
+            // using copyObject only if copying object from one external
+            // backend to the same external backend
```

Comment:
Do we check and handle if someone requests to copy object from one ext. backend to a different one? E.g. return an error?

Reply:
if copy object from one ext. backend to a different one, we do a GET and then a PUT 
PS: condition here: https://github.com/scality/S3/pull/897/files#diff-5ce9bf7b97556522975c2fae1a6944efR361